src/lib.rs

```diff
@@ -121,7 +121,7 @@ mod protocol;
 pub mod services;
 mod session;
 #[cfg(feature = "stream")]
-mod stream;
+pub mod stream;
```

Comment:
Oh wow, I somehow just completely missed it, even when looking at the generated docs.  
No wonder I had the feeling when making that change that it was weird that this would have been left unnoticed, no idea what made me miss that re-export.  
I guess I can therefore close the PR and simply rewrite the corresponding `use` statements in the `rust-openstack` PR.